rubyforgood/diaper
# IN PROGRESS Barcode ajax

To test it:

Go to /donations/new - put the mouse in the really wide field and enter a barcode string (a bunch of numbers) and press enter. Alternately just use an actual barcode reader.

Watch the console log for output.

Weird behavior:
The first time you try using it, it won't do anything, but the second time (when it shows error messages) it will work. Not sure why it's doing this.

the `data` object that gets returned *should* have the `@barcode_item.to_json` in it, but I don't see that coming up in the inspector.

TODO:
wire up success / failure functions to hook into the workflow. Ideally, this whole process should be generalized enough that we can drop it in for `Donation`, `Distribution`, `Transfer`, `Adjustment`s
